src/vm/eventpipesession.cpp

```diff
@@ -30,7 +29,6 @@ EventPipeSession::EventPipeSession(
     m_circularBufferSizeInBytes = circularBufferSizeInMB * 1024 * 1024; // 1MB;
     m_rundownEnabled = false;
     m_pProviderList = new EventPipeSessionProviderList(pProviders, numProviders);
-    m_multiFileTraceLengthInSeconds = multiFileTraceLengthInSeconds;
```

Comment:
I'm fine if moving this makes it easier at the moment, but I expect once we have multi-session support you will need to re-instate it here.

Reply:
I'm not convinced this belong to the session itself, but to the way the "streams" will be flushed. Specially in the multiclient scenario.